src/AI/WallyAI.java

```diff
@@ -692,7 +692,7 @@ private GameAction findTravelAction(
     boolean ignoreResident = true;
     ArrayList<XYCoord> destinations = Utils.findPossibleDestinations(unit, gameMap, ignoreResident);
     if( ignoreSafety ) // If we *must* travel, make sure we do actually move.
-      destinations.remove(0);
+      destinations.remove(new XYCoord(unit.x, unit.y));
```

Comment:
You're saying [this](https://docs.oracle.com/javase/8/docs/api/java/util/ArrayList.html#remove-int-) doesn't work?

Reply:
It works just fine. It even still works for that use case, I think - I'm pretty sure I misdiagnosed which piece of code produced the "sit in place when you're not supposed to" behavior.
That said, this is more explicit and doesn't depend on the implementation details (i.e. ordering) of `findPossibleDestinations()`, so I see no reason to change it back.